internal/commands/profiling/asyncprofiler/getTaskProgress.go

```diff
@@ -0,0 +1,36 @@
+package asyncprofiler
+
+import (
+	"github.com/apache/skywalking-cli/pkg/display"
+	"github.com/apache/skywalking-cli/pkg/display/displayable"
+	"github.com/apache/skywalking-cli/pkg/graphql/profiling"
+	"github.com/urfave/cli/v2"
+)
+
+var getTaskProgressCommand = &cli.Command{
+	Name:    "progress",
+	Aliases: []string{"logs", "p"},
```

Comment:
Can it be used as `swctl profiling asyncprofiler logs`? I think it's a difference in the meaning of progress.